### DISCUSSION TITLE
Are there plans for the 1.20.2 version?

### DISCUSSION THREAD

**Discussion by OscarQuintanilla** [vectorwing/FarmersDelight#829] [Q&A]
I am starting a new world in this version and would like to know if there is going to be a port for it since I really love this mod.

**Comment by vectorwing** [comment]
Hello!

I'm taking an extended break from modding to recover my batteries. I'll try to sort out a 1.20.2 port eventually, but no date confirmed yet. Sorry! ^^

**Comment by nuypham** [comment]
I'm on 1.20.2 also. Hope this get update to 1.20.2 soon.
